options/locale/locale_en-US.ini

```diff
@@ -1296,6 +1296,12 @@ pulls.update_not_allowed = You are not allowed to update branch
 pulls.outdated_with_base_branch = This branch is out-of-date with the base branch
 pulls.closed_at = `closed this pull request <a id="%[1]s" href="#%[1]s">%[2]s</a>`
 pulls.reopened_at = `reopened this pull request <a id="%[1]s" href="#%[1]s">%[2]s</a>`
+pulls.merge_instruction_hint = You can also view
+pulls.merge_instruction_link_text = command line instructions
+pulls.merge_instruction_step1 = Step 1: 
```

Comment:
This could be moved to top-level, e.g. `step1`

Reply:
What did you mean? Rename it to `pulls.step1`?

Reply:
No, without any section, e.g. like line 1-89 in the file. It's generic.